Sources/KindeSDK/Auth/Auth.swift

```diff
@@ -79,6 +171,32 @@ public final class Auth {
         return params[key] ?? nil
     }
     
+    public func getPermissions(options: ApiOptions? = nil) async throws -> Permissions {
+        if options?.forceApi == true {
+            let response = try await PermissionsAPI.getPermissions()
+            guard response.success, let data = response.data else {
+                throw NSError(domain: "KindeSDK", code: -1, userInfo: [NSLocalizedDescriptionKey: "Failed to fetch permissions from API - check network and authentication"])
+            }
+            let orgCode = data.orgCode ?? ""
+            let organization = Organization(code: orgCode)
+            return Permissions(organization: organization, permissions: response.getPermissionKeys())
+        } else {
+            if let permissionsClaim = getClaim(forKey: ClaimKey.permissions.rawValue),
+               let permissionsArray = permissionsClaim.value as? [String],
+               let orgCodeClaim = getClaim(forKey: ClaimKey.organisationCode.rawValue),
+               let orgCode = orgCodeClaim.value as? String {
+                
+                let organization = Organization(code: orgCode)
+                let permissions = Permissions(organization: organization,
+                                              permissions: permissionsArray)
+                return permissions
+            }
```

Comment:
Since the same logic is in the `getPermissions()` function, we can use that directly here. 

```suggestion
        if let permissions = getPermissions() {
            return permissions
        }
```